modules/mobile-agents/android/jax-rs/src/main/java/org/wso2/carbon/mdm/services/android/OperationMgtService.java

```diff
@@ -708,8 +716,8 @@ public Response encryptStorage(@HeaderParam("Accept") String acceptHeader,
     @POST
     @Produces(MediaType.APPLICATION_JSON)
     @Path("change-lock-code")
```

Comment:
As we are ideally creating REST resources, better to use a noun for the path rather than a verb.